### ISSUE TITLE
Compiling a script and AHKS losing the line it was on

### ISSUE THREAD

**Issue by noticz** [maestrith/AHK-Studio#57]
You fixed the issue with Compile_AHK.exe for saving comments inside the script....thanks for that!

There seems to be a new issue though when compiling a script. When I compile a script with Compile_AHK.exe it seems to compile okay and then once compiling is done it jumps to the very top of the script. I pretty much lose the line that I am working on.

Wondering if AHKS knows that the file was edited so it moves to the edited line....is there an option for this?

**Comment by maestrith** [comment]
Please post a gif to show what you are doing and what Studio is doing so I can see and hopefully understand a bit better.

**Comment by noticz** [comment]
In the screen shot the only buttons I press is to compile, then I click enter and after it is done compiling this is what happens....It jumps to the top of the script.
![2016-01-07_14-27-37](https://cloud.githubusercontent.com/assets/11874318/12183303/50a6efcc-b54b-11e5-9bbb-4f661c917026.gif)

**Comment by noticz** [comment]
This feature was fixed with an update, thanks maestrith
